test/Test/Orphans.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE StandaloneDeriving #-}
+module Test.Orphans where
+
+import HieDb.Types
+import Module (ModuleName, moduleName, moduleNameString, moduleUnitId)
+import Name (Name, nameModule, nameOccName)
+import OccName (OccName, occNameString)
+
+instance Show ModuleName where show = moduleNameString
```

Comment:
I'm a bit unsure about how to approach testing with these types.
Most hspec combinators (like shouldBe, shouldReturn) depends on Show and Eq instance, but most of these GHC types don't have one. Is it ok to derive bunch of orphan instances just for the purpose of testing, or how do GHC devs approach this?

Reply:
Try projecting out parts which have the instances? Like `moduleNameString`.

If this proves to be insufficient, you can define orphans in the test suite.